Resources/public/js/views/services/ez-serversideviewservice.js

```diff
@@ -161,6 +168,25 @@ YUI.add('ez-serversideviewservice', function (Y) {
         },
 
         /**
+         * Fires notify event basing on node
+         *
+         * @method _responseNotify
+         * @protected
+         * @param {Node} node
+         */
+        _responseNotify: function (node) {
+            console.log(node);
+            this.fire('notify', {
+                notification: {
+                    text: node.getContent(),
+                    identifier: Math.random(),
+                    state: node.getAttribute('data-state'),
+                    timeout: 0,
```

Comment:
the timeout should be set to 0 only on error notification, for the others you can set it to 5 (we should introduce a constant for that at some point...)